### DISCUSSION TITLE
Configure session timeout for login with username and password

### DISCUSSION THREAD

**Discussion by cm-schl** [semaphoreui/semaphore#2391] [Q&A] [answered]
Hi,

is there any possibility to configure the timeout when a login is done by username & password? Actually semaphore UI seams to never log out the users at the moment.

Best regards

**Comment by fiftin** [comment, marked as answer]
Currently it is not possible. I will create issue for your question.

**Reply by cm-schl** [reply]
Thanks!

**Comment by fiftin** [comment]
https://github.com/semaphoreui/semaphore/issues/2403
